app/serializers/api/v1/package_serializer.rb

```diff
@@ -3,7 +3,7 @@ class PackageSerializer < ApplicationSerializer
     embed :ids, include: true
 
     has_one :package_type, serializer: PackageTypeSerializer
-    has_many :images, serializer: ImageSerializer, root: :package_images
+    has_many :images, serializer: ImageSerializer
```

Comment:
Will this have any effect on other apps??

Reply:
The reason for that is that Stock mostly use `StockItemSerializer` and browse expects `BrowsePackageSerializer`

Reply:
part of the Serializer hell I'd like to fix haha